README.md

```diff
@@ -74,6 +74,49 @@ You can run individual actions like this:
 v my-project app bash
 ```
 
+### Adding more advanced actions
+You can always use the feature set of the `sh` shell. Like operators, conditions and variables. So you can build powerful workflows which saves you brain capacity and time.
+```yaml
+services:
+  app:
+    actions:
+      # Shell Operators
+      #
+      # use yaml multiline strings and shell operators to run multiple
+      # commands after the other.
+      start: >
+        docker-compose up -d &&
+        echo "startup done" &&
+        docker-compose exec app npm run watch &&
+        code .
+
+      # Shell Variables
+      #
+      # $@ is a shell variable which holds all passed arguments
+      # so it is possible to pass all arguments to a command by running:
+      # v my-project app exec npm install double-vue
+      exec: docker-compose exec app $@
+      # You can pass individual shell arguments as well by using $1..$[x]
+      create-branch: git checkout -b feature/$1
+
+      # Custom Shell
+      #
+      # If running your commands in a simple `sh` is not enough you could
+      # launch your favorite shell as well to leverage more complex tasks
+      watch: zsh -c "source ~/.zshrc && nvm use && npm run watch"
```

Comment:
Seems like it would be kinda nice to be able to configure the shell that should be used to start commands. Should be fairly easy, no?

Reply:
yeah, kind of. I think we have to configure the paths/names of the rc files for several popular shells. 
But I think the via actions could get way slower because sourcing large/complex rc files is not for free. At the moment I would leave the trade-off to the user even it is more complicated to opt-in. But if we would source the files automatically we should have to design an opt-out.

Reply:
Yeah, I though of adding options to opt in to a specific shell on different levels, e.g. globally (in a new via.yaml), or on project level, or on service level